testgrid/config/yaml2proto/testdata/NoTestgroup.yaml

```diff
@@ -0,0 +1,28 @@
+default_test_group:
```

Comment:
Please use TestGroup or Testgroup consistently. Given default_test_group I'd expect this to be test-group